react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class RegistrationForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      password: ''
+    };
+
+    this.handleUsernameChange = this.handleUsernameChange.bind(this);
+    this.handlePasswordChange = this.handlePasswordChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleUsernameChange(event) {
+    this.setState({ username: event.target.value });
+  }
+
+  handlePasswordChange(event) {
+    this.setState({ password: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    console.log('State:', this.state);
+  }
+
+  render() {
+    return <form>
+      <label>Username:</label>
+      <input value={this.state.username} onChange={this.handleUsernameChange}></input>
```

Comment:
`input` elements are self closing elements, so shouldn't have a closing tag, and `input`s should have a `name` attribute.

```JSX
<input name="username" type="text" value={this.state.username} onChange={this.handleUsernameChange} />
```